test/Twig/Tests/Fixtures/functions/recursive_block_with_inheritance.test

```diff
@@ -12,7 +12,7 @@
 {% block list %}<ul>{{ block('children') }}</ul>{% endblock %}
 {% block children %}{% set currentItem = item %}{% for item in currentItem %}{{ block('item') }}{% endfor %}{% set item = currentItem %}{% endblock %}
 {% block item %}<li>{% if item is not iterable %}{{ block('label') }}{% else %}{{ block('list') }}{% endif %}</li>{% endblock %}
-{% block label %}{{ item }}{{ block('unknown') }}{% endblock %}
```

Comment:
the case of an unknown block must be kept in a separate test marked as legacy, to ensure we don't break its support in 1.x